Packs/Campaign/Scripts/ShowCampaignLastIncidentOccurred/ShowCampaignLastIncidentOccurred.py

```diff
@@ -44,10 +44,11 @@ def get_last_incident_occurred(incident_ids) -> str:
 def main():
 
     try:
-        incident_ids = get_incident_ids()
-        last_incident_occurred = get_last_incident_occurred(incident_ids)
-
-        if last_incident_occurred:
+        if (
+            (incident_ids := get_incident_ids())
+            and (last_incident_occurred := get_last_incident_occurred(incident_ids))
+        ):
+            last_incident_occurred = get_last_incident_occurred(incident_ids)
             html_readable_output = f"<div style='text-align:center; font-size:17px; padding: 15px;'>" \
                                    f"Last Incident Occurred</br> <div style='font-size:24px;'> " \
                                    f"{last_incident_occurred} </div></div>"
```

Comment:
since only the end changes, consider removing the rest of the html code, which is repeated twice